### PR TITLE
Broaden Choice Uniqueness

### PR DESCRIPTION
**Pull request by joeljfischer**
This proposal broadens choice and sub-menu uniqueness to allow `PerformInteraction` and sub-menu `AddCommand` primary text to be identical.

### CODE REVIEW

**Comment by joeljfischer** [comment]
@theresalech This is ready for review

File: proposals/nnnn-broaden-choice-uniqueness.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Broaden Choice Uniqueness
+
+* Proposal: [SDL-NNNN](NNNN-broaden-choice-uniqueness.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android]
+
+## Introduction
+
+This proposal broadens choice and sub-menu uniqueness to allow `PerformInteraction` and sub-menu `AddCommand` primary text to be identical.
```
Comment (theresalech) [inline proposals/nnnn-broaden-choice-uniqueness.md]: This is different from the PR description.  Can you please update accordingly?

**Comment by yang1070** [comment]
the text of menu item or choice item is also used for voice commands, which has the limitation that requires the text to be unique under a give context. That might be a factor to consider.

**Comment by joeljfischer** [comment]
@theresalech fixed

**Pull request merged**